dolphinscheduler-ui/src/js/conf/home/pages/security/pages/tenement/_source/createTenement.vue

```diff
@@ -146,6 +146,13 @@
           this.$message.warning(`${i18n.$t('Please enter name')}`)
           return false
         }
+        // Verify tenant name cannot contain special characters
+        let isSpecial = /[~#^$@%&!*()<>《》:;'"{}【】	]/gi
+        if (isSpecial.test(this.tenantName)) {
+          this.$message.warning(`${i18n.$t('Please enter tenant name without special characters')}`)
+          return false
+        }
```

Comment:
i think we should keep the front and back consistent, the api module should also limit the tenant name.
on the other hand, should we limit tenant name just 'number and character'?